backend/src/domain/error.ts

```diff
@@ -0,0 +1,20 @@
+export const DatabaseMigrationError = 'DATABASE_MIGRATION is optional, but if set must be one of [UP, DOWN]';
+export const DatabaseConnectionError = 'Missing environment variables for database connection';
+export const EntryNotFoundError = (tableName: string, filter: any) =>
+  `Unable to find entry in ${tableName} with filter [${JSON.stringify(filter)}]`;
+export const InsertError = (resultLength: number) => `Insert expected one element but was ${resultLength}`;
+export const MealNotFoundError = (mealName: string) => `Couldn't find meal with name:  ${mealName.toUpperCase()})`;
+export const Level60Error = 'Somehow level 60 produced ingredients not populated for buddy-combination-dao';

```

Comment:
Regarding the following errors:
- `Level60Error`
- `EnergyPercentageError`
- `LevelError`
- `NatureError`

These could all be replaced by an error called for example `StatError` and be moved into `domain/error/stat/stat-error.ts`